src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -158,8 +162,19 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
             // Print a legend message for auto-reference markers used
             if (autoReferenceFound)
             {
-                Console.WriteLine(Strings.ListPkg_AutoReferenceDescription);
+                reportRenderer.WriteLine(Strings.ListPkg_AutoReferenceDescription);
+
+                // Should log into json output ?
```

Comment:
`autoReferenceFound` is package referenced from sdk.
It's something we didn't consider for spec, so I far I haven't seen actual example of this.

Reply:
Test a project that targets netstandard2.0. It should have an implicit package reference.